src/components/search-form.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { FIELD_NAMES } from './user';
+
+import UserInfo from '../stores/user-info';
+import UserAction from '../actions/user-action';
+
+export default class SearchForm extends React.Component {
+    constructor (props) {
+        super(props);
+
+        this.state = {
+            user: new UserInfo()
```

Comment:
we should rename `user-userInfo` here and in `findUser` action and in store's handler